cli/command/cli.go

```diff
@@ -271,6 +276,27 @@ func newHTTPClient(host string, tlsOptions *tlsconfig.Options) (*http.Client, er
 	}, nil
 }
 
+func tlsConfigFromOptions(opts *cliflags.CommonOptions) (*tlsconfig.Options, error) {
+	if opts.TLSOptions == nil {
+		return nil, nil
+	}
+
+	tlsMinVersion, err := dopts.ParseTLSMinVersion(opts.TLSOptions.MinVersion)
+	if err != nil {
+		return nil, err
+	}
+
+	tlsOptions := &tlsconfig.Options{
+		CAFile:             opts.TLSOptions.CAFile,
+		CertFile:           opts.TLSOptions.CertFile,
+		KeyFile:            opts.TLSOptions.KeyFile,
+		InsecureSkipVerify: !opts.TLSVerify,
```

Comment:
I guess it looks like it's an existing option so probably not much we can do to fix it :(